src/common/models/Lockfile.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: define lockfile type better
```

Comment:
Worth making an issue?

Reply:
Thanks for the suggestion, done! https://github.com/conda-incubator/conda-store-ui/issues/413